Dockerfile.template

```diff
@@ -9,6 +9,7 @@ RUN apt-get update && \
         libffi-dev \
         libssl-dev \
         matchbox \
+        nano \
```

Comment:
Nooooo :) 
Never put an editor in a Dockerfile.

Reply:
lol how did this even get here.. these were some of the tests dockerfile lines i was using messing with the container itself .. ugh i must of pressed the wrong thing on my forked repo.. sorry now I have to learn how to only edit on directly this branch